src/main/java/com/gluonhq/substrate/model/InternalProjectConfiguration.java

```diff
@@ -235,10 +235,14 @@ public boolean useCustomJavafxStaticLibs() {
     }
 
     public Path getDefaultJavafxStaticPath() {
+        Triplet targetTriplet = getTargetTriplet();
+        if (new Triplet(Constants.Profile.IOS_SIM).equals(targetTriplet)) {
+            targetTriplet = new Triplet(Constants.Profile.IOS);
```

Comment:
This means the arch is removed from the triplet, but it is used later at https://github.com/gluonhq/substrate/pull/975/files#diff-1e93749c0a14ab381e919971325d7c981813fdc8799bf48a1db9ee7cb641ed6bR374

Is it removed because the download doesn't deal with arch in its name?

Reply:
We don't remove the arch but simply create a new triplet just so we can get the arch from iOS-AArch64. 
This is required because we don't publish static JavaFX for iOS-x86_64, as we can use the AArch64 version.

Reply:
it sounds confusing we download an AArch64 artifact because that also works on AMD64.
The AMD64 artifact probably uses fat libraries? In that case, it will work, but it's very confusing. We should probably either remove the architecture from the artifact, or have 2 artifacts. But that is tbd in CI.